$MirOS: ports/misc/mc/patches/patch-src_util_h,v 1.1.7.1 2005/03/18 15:50:29 tg Exp $
--- src/util.h.orig	Thu Jan 13 19:20:25 2005
+++ src/util.h	Mon Feb 20 18:32:48 2006
@@ -193,9 +193,9 @@ GList *list_append_unique (GList *list, 
 /* Position saving and restoring */
 
 /* file where positions are stored */
-#define MC_FILEPOS ".mc/filepos"
+#define MC_FILEPOS ".etc/mc/filepos"
 /* temporary file */
-#define MC_FILEPOS_TMP ".mc/filepos.tmp"
+#define MC_FILEPOS_TMP ".etc/mc/filepos.tmp"
 /* maximum entries in MC_FILEPOS */
 #define MC_FILEPOS_ENTRIES 1024
 /* Load position for the given filename */
